azure-stack/hci/upgrade/install-enable-network-atc.md

```diff
@@ -41,7 +41,7 @@ Before you install and enable Network ATC on your existing Azure Stack HCI, make
 ## Steps to install and enable Network ATC
 
 > [!IMPORTANT]
-> If you don't have running workloads on your nodes, just add your intent command as if this was a new cluster. You don't need to continue with the next set of instructions.
+> If you don't have running workloads on your nodes, execute [Step 4: Remove the existing configuration on the paused node without running VMs](#step-4-remove-the-existing-configuration-on-the-paused-node-without-running-vms) to remove any previous configurations that could conflict with Network ATC, then add your intent(s) following the standard procedures found in [Deploy host networking with Network ATC](https://learn.microsoft.com/en-us/azure-stack/hci/deploy/network-atc?tabs=22H2)
```

Comment:
@helmste - I will need to check with the PM on this suggested change.

Reply:
@ronmiab - if you attempt to add an intent and conflicting settings exist on the cluster, this will cause an error.  The only way to avoid these conflicts is to ensure all possible conflicting settings are removed then add the intent.